mkdocs.yml

```diff
@@ -0,0 +1,2 @@
+site_name: My Docs
+site_url: https://example.com/
```

Comment:
what's that for?

Reply:
Removed. It was needed for `readthedoc.io` but we decided not to use that.